docs/introduction.rst

```diff
@@ -1,73 +1,57 @@
 
-Data Items are displayed using one or more *Display Items*.
+If a data item includes complex components, it will need to be reduced into a scalar image or line plot in order to be displayed. For more info, see :ref:`data-items`.
 
-See :ref:`data-items`.
+Sometimes, data items can be part of a larger 'stack' of data which will require higher level organization that the data panel can provide. See :ref:`data-items` for more info.
 
-Projects
---------
-Data items are organized into projects. Projects are stored on disk and take the form of an index file and an associated data directory.
+The INFO PANEL shows you the location and value of the data in a display panel under your cursor. You can choose how you would like the :ref:`data-items` (coordinate system) to be defined in the inspector panel.
 
-You can have any number of projects, but only a single one can be open at once. When you open a project, it is associated with a main document window.
+Click on any data item, display panel, or graphic to select it. You can select more than one item of the same kind using [ctrl + click] on Windows or [cmd + click] on macOS. 
 
-See :ref:`data-management`.
+Selecting more than one display panel will display with a solid blue rectangle around the primary selection and with dashed lines around the secondary selections. If a computation or function can only use one input, the primary selection will be used. For computations using multiple inputs, secondary selections may be used as well.
 
-Workspaces
-----------
-The main window is organized around a main area called a workspace. A workspace has a layout which includes one or more display panels in which display items can be visualized.
+Selecting multiple data items will allow you to move them at the same time. However, multiple data items cannot be dragged into the same display panel.
 
-You can create additional workspace for different uses and easily switch between them.
+Selecting multiple graphics in the same display panel will allow you to move them at the same time. The inspector panel will display options for all of the selected graphics.
 
-In addition, the main window has a variety of utility panels attached or associated with the main window. Some examples of utility panels are the data browser panel, the inspector panel, and the info and histogram panels.
+Graphics and Masks
+==================
+Nion Swift provides plenty of useful graphics to narrow down your view of the data. Use graphics to highlight portions of the data, select areas for processing, or to focus on specific sections of data.
 
-See :ref:`user-interface`.
+Masking allows you to block out the rest of an image and see only what you want. Choose from a variety of mask shapes to precisely define your area of interest.
```

Comment:
... FYI not only "see" but use in processing. i.e. you might want to only set the masked portion of an image to 0's. You can probably come up with a better wording of my explanation :)